### DISCUSSION TITLE
更新2.1.1后，文件夹里面有ipa文件的时候 点开网页变成空白

### DISCUSSION THREAD

**Discussion by xiaowang7212** [AlistGo/alist#682] [General]
更新2.1.1后，文件夹里面有ipa文件的时候 点开网页变成空白

**Comment by xhofe** [comment]
已修复。

**Reply by xiaowang7212** [reply]
请问 我改怎么修复他

**Reply by xhofe** [reply]
更新

**Reply by xiaowang7212** [reply]
后台更新了 然后第一次打开没有问题 刷新后 还是这样

**Reply by xhofe** [reply]
测试已经没有问题。https://alist.xhofe.top/SharePoint/%E4%B8%AD%E6%96%87%E6%B5%8B%E8%AF%95.ipa

**Reply by xiaowang7212** [reply]
好了 可以了 更新到2.1.2解决了，谢谢

**Reply by zhlx2835** [reply]
我最近安装的v3，想在线安装ipa，需要怎么配置，有没有说明教程，现在直接安装是白图标也打不开。
